### PR TITLE
OCPBUGS-6007: OVN-Kubernetes doc clarification

### PR DESCRIPTION
**Pull request by ahardin-rh**
Issue:
https://issues.redhat.com/browse/OCPBUGS-6007

Version(s):
4.12+

Link to docs preview:
https://54884--docspreview.netlify.app/openshift-enterprise/latest/networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.html

### CODE REVIEW

**Review by mffiedler — APPROVED** [review]
LGTM

File: modules/nw-ovn-kubernetes-features.adoc [hunk 1]
```diff
@@ -2,14 +2,13 @@
 //
 // * networking/ovn_kubernetes_network_provider/about-ovn-kubernetes.adoc
 
-[id="nw-ovn-kubernetes-features_{context}"]
-= OVN-Kubernetes features
+[id="nw-ovn-kubernetes-purpose_{context}"]
+= OVN-Kubernetes purpose
 
-The OVN-Kubernetes network plugin implements the following features:
+The OVN-Kubnernetes network plugin is an open-source, fully-featured Kubernetes CNI plugin that uses Open Virtual Network (OVN) to manage network traffic flows. OVN is a community developed, vendor-agnostic network virtualization solution. The OVN-Kubnernetes network plugin:
```
Comment (dcbw) [inline modules/nw-ovn-kubernetes-features.adoc:8]: @ahardin-rh how about at the end "The OVN-Kubernetes network plugin provides the following advantages over OpenShift SDN:"

And then we change the list just below to something like:

- Full support for IPv6 single-stack and IPv4/IPv6 dual-stack networking on supported platforms
- Support for hybrid clusters with both Linux and Microsoft Windows workloads
- Optional IPsec encryption of intra-cluster communications
- Offload of network data processing from host CPU to compatible network cards and Data Proccessing Units (DPUs)

Or something like that... eg pull out the 3-4 most important differences between OVN-Kube and OpenShift SDN to give customers a better idea of why we're making OVN-Kube the default in 4.12.

What do you think?

**Review by GroceryBoyJr — COMMENTED** [review]
LGTM

**Comment by ahardin-rh** [comment]
cherrypick enterprise-4.13

File: modules/nw-ovn-kubernetes-features.adoc [hunk 2]
```diff
@@ -2,14 +2,20 @@
...
```
Comment (bergerhoffer) [inline modules/nw-ovn-kubernetes-features.adoc:8]: @ahardin-rh This line introduced 2 instances of a typo "Kubnernetes". Would you mind submitting a followup PR to fix it?

**Pull request merged**